.codesandbox/ci.json

```diff
@@ -11,7 +11,8 @@
     "/examples/starter-algolia",
     "/examples/starter",
     "/examples/reshape",
-    "/examples/vue"
+    "/examples/vue",
+    "/examples/vue-instantsearch"
```

Comment:
**[suggestion]** We're limited to how many sandboxes we can have built automatically on every PR, so I think we can leave it out. Normally it _should_ still show up in this PR.